openedx/core/djangoapps/programs/signals.py

```diff
@@ -50,5 +50,6 @@ class of the object instance that sent this signal
         status,
     )
     # import here, because signal is registered at startup, but items in tasks are not yet able to be loaded
-    from openedx.core.djangoapps.programs.tasks.v1.tasks import award_program_certificates
+    from openedx.core.djangoapps.programs.tasks.v1.tasks import award_course_certificates, award_program_certificates
     award_program_certificates.delay(user.username)
+    award_course_certificates.delay(user.username, course_key)
```

Comment:
This works for awarded certs. But what about other cert changes like revocations? Looks like that signal only gets sent for generating/downloadable.

Reply:
Ah right on, missed that. I can do this pretty easily by just creating a second signal and a second signal handler, and overwrite the save on to send that signal for anything, sound fair?

Reply:
Yeah that's probably fine. There are fields we don't care about the changes on? So maybe just signal on status changes? I dunno if that's worth optimizing on though. But yeah, a second signal is probably needed.